addon/templates/components/course-overview.hbs

```diff
@@ -14,10 +14,13 @@
     {{#link-to "print_course" course (query-params unpublished=true) class="print"}}
       {{fa-icon "print" title=(t "general.printSummary") fixedWidth=true}}
     {{/link-to}}
-    {{#if (and (is-fulfilled showRollover) (await showRollover))}}
-      {{#link-to "course.rollover" course class="rollover"}}
-        {{fa-icon "random" title=(t "general.courseRollover") fixedWidth=true}}
-      {{/link-to}}
+    {{#if (and (is-fulfilled this.showRollover) (await this.showRollover))}}
+      <a
```

Comment:
use the following instead: 
```hbs
      <span {{action "transitionToRollover"}} class="clickable">
        {{fa-icon "random" fixedWidth=true title=(t "general.courseRollover")}}
      </span>
```
this is more in line with our existing patterns. will require minor style tweak in `course-overview.scss`

```scss
  .course-overview-actions {
    @include ilios-overview-actions;

    a, span {
      color: $header-blue;
      font-size: 1.2rem;
      margin-right: .5rem;
    }
  }
```

Reply:
👍